rest-tests/src/test/java/io/syndesis/qe/rest/tests/integrations/S3Steps.java

```diff
@@ -42,12 +46,20 @@ public S3Steps() {
 
     @Given("^create S3 polling START action step with bucket: \"([^\"]*)\"$")
     public void createS3PollingStep(String bucketName) {
+        createS3PollingStep(bucketName, null);
+    }
+
+    @Given("^create S3 polling START action step with bucket: \"([^\"]*)\" and prefix \"([^\"]*)\"$")
+    public void createS3PollingStep(String bucketName, String prefix) {
         final Connector s3Connector = connectorsEndpoint.get("aws-s3");
         final Connection s3Connection = connectionsEndpoint.get(S3BucketNameBuilder.getBucketName(bucketName));
         final Action s3PollingAction = TestUtils.findConnectorAction(s3Connector, "aws-s3-polling-bucket-connector");
         final Map<String, String> properties = TestUtils.map(TestUtils.map("deleteAfterRead", "false",
                 "maxMessagesPerPoll", "10",
                 "delay", "1000"));
+        if (prefix!=null){
```

Comment:
run some kind of auto-layout on your changes so we have a nice code: (prefix != null)